### PR TITLE
README and img files

### PR DESCRIPTION
**Pull request by Farah202324**
README and img files editing the change you requested + adding the build file + completing the Associations file +editing some spelling errors in other files + one of u missed adding optional for the id and some of your seed data were not compatible with the interface same with the models so I had to fix them all  + do not worry because the number of file changes it is just the images skip them please. 
thanks for your time .

### CODE REVIEW

**Review by momenqudaih — APPROVED** [review]

**Review by momenmarwan — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+- [Introduction](#Menu)
+- [Features](#features)
+- [ UserStory ](#User)
+- [Database Schema](#Database)
+- [Technologies Used](#Technologies)
+- [Demo](#Demo)
+- [Figma Design](#Figma)
+- [Contributing](#contributing)
+- [Acknowledgements](#Acknowledgements)
+# Features
```
Comment (moustf) [inline README.md:13]: Name the features as User Stories, and each story needs to start with `As a user, I should be able to ...`.

E.g.: As a restaurant owner user, I should be able to register and authenticate my staff.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+- [Acknowledgements](#Acknowledgements)
+# Features 
+- User registration and authentication for the staff.
+- Menu management for the restaurant.
+- Order placement by customers.
+- Real-time order tracking.
+- Notification system for order updates.
+- Admin dashboard for managing orders and menus.
+- supports English and Arabic language.
+# User Story
```
Comment (moustf) [inline README.md:21]: This should e the user journey.

User journey illustrated the journey of our client on the website.
E.x.: When the user goes to a restaurant to eat, the user will find a QR taped to each table, after scanning this QR, our website will be opened and the user can order from the menu, (and you complete this way).

**Review by moustf — CHANGES_REQUESTED** [review]
Really pretty shaping, just added a few notes.
Thank you, Farah!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,88 @@
...
+#### staff :
+<img src=./img/signup1.png style='height:150px'>  <img src=./img/login.png style='height:150px'>
+
+<img src=./img/dashboard.png style='height:150px'> <img src=./img/kitchen.png style='height:150px'>
+
+# Contributing
+Contributions are welcome! If you find any issues or have suggestions for improvements, feel free to open an issue or submit a pull request.
+
+# coders 💻
+#### team:
```
Comment (momenqudaih) [inline README.md:73]: add mentions like that @Farah202324  plz

**Review by momenqudaih — APPROVED** [review]
Good job farah 😍💪

**Review by momenmarwan — APPROVED** [review]

**Pull request closed without merging**